cli/Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM alpine:3.5
+FROM alpine:3.6
 MAINTAINER Kontena, Inc. <info@kontena.io>
 
-ADD https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 /usr/local/bin/docker
 ARG CLI_VERSION
 
 RUN apk update && \
   apk --update add ruby ruby-json ruby-bigdecimal ruby-io-console \
-  ca-certificates libssl1.0 openssl libstdc++ && \
+  curl ca-certificates libssl1.0 openssl libstdc++ && \
+  curl -sL https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz > /tmp/docker.tgz && \
```

Comment:
Could be simplified to just `curl ... | tar xz`, leaving the `/tmp/docker.tgz` out.